packages/debugger/src/panels/variables/tree.tsx

```diff
@@ -452,57 +454,57 @@ const VariableComponent = (props: IVariableComponentProps): JSX.Element => {
   };
 
   return (
-    <li
-      onClick={(e): Promise<void> => onVariableClicked(e)}
-      onMouseDown={e => {
-        e.stopPropagation();
-        onSelection(variable);
-      }}
-      onMouseOver={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
-        if (onHoverChanged) {
-          onHoverChanged({ target: event.currentTarget, variable });
-          event.stopPropagation();
-        }
-      }}
-      onMouseLeave={(event: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
-        if (onHoverChanged) {
-          onHoverChanged({
-            target: event.relatedTarget as EventTarget & HTMLElement,
-            variable: null
-          });
-          event.stopPropagation();
-        }
-      }}
-    >
-      <span
-        className={
-          'jp-DebuggerVariables-collapser' +
-          (expanded ? ' jp-mod-expanded' : '')
-        }
+      <TreeItem
+        onClick={(e): Promise<void> => onVariableClicked(e)}
```

Comment:
The expanded property should be used to set the attribute on the tree item

```suggestion
        expanded={expanded}
        onClick={(e): Promise<void> => onVariableClicked(e)}
```